### ISSUE TITLE
[Rule change] `avoid-late-keyword` - option to only error when `late` variable is initialized

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#1113]
**What rule do you want to change?**
`avoid-late-keyword`

**How will the change be implemented? (New option, new default behavior, etc.)?**
new option

**Please provide some example code that this change will affect:**
```dart
late bool foo;
late bool bar = false;
```

**What does the rule currently do for this code?**
error on both lines

**What will the rule do after it's changed?**
only error on `bool`

**Are you willing to submit a pull request to implement this change?**
if it's easy

**Comment by incendial** [comment]
Available in 5.5.0 🚀

**Comment by DetachHead** [comment]
@incendial i'm sorry i worded this issue very badly. i meant i wanted an option to ban `late` variables that are initialized (as in, the `late` keyword isn't necessary). but thinking about it more i feel it should probably be a new rule anyway. i will raise a new issue
